JMS/activemq/docker-compose.yml

```diff
@@ -6,4 +6,4 @@ services:
       - "8000:8000"
     hostname: "kaazing.example.com"
   activemq:
-    build: broker
+    image: webcenter/activemq:latest
```

Comment:
I've filed a new [issue] (https://github.com/kaazing/tickets/issues/702) for the above mentioned behavior.